backend/de.metas.manufacturing/src/main/java/org/eevolution/productioncandidate/model/interceptor/PP_Order_Candidate.java

```diff
@@ -156,6 +150,19 @@ public void checkMaturingCandidate(@NonNull final I_PP_Order_Candidate ppOrderCa
 		}
 	}
 
+	@ModelChange(
+			timings = { ModelValidator.TYPE_BEFORE_NEW, ModelValidator.TYPE_BEFORE_CHANGE },
+			ifColumnsChanged = { I_PP_Order_Candidate.COLUMNNAME_PP_Order_Candidate_Parent_ID, I_PP_Order_Candidate.COLUMNNAME_SeqNo })
+	public void checkSeqNo(@NonNull final I_PP_Order_Candidate ppOrderCandidateRecord)
```

Comment:
what's the actual user story for this logic?

Reply:
we are going to handle this topic in a follow-up issue